api/openapi.yaml

```diff
@@ -805,6 +988,225 @@ paths:
       - bearerAuth: []
       tags:
       - sharing
+  /user:
+    get:
+      description: Get a list of (local) users.
+      operationId: listUsers
+      parameters:
+      - explode: true
+        in: query
+        name: limit
+        required: false
+        schema:
+          type: integer
+        style: form
+      - explode: true
+        in: query
+        name: offset
+        required: false
+        schema:
+          type: integer
+        style: form
+      - explode: true
+        in: query
+        name: count
+        required: false
+        schema:
+          type: boolean
+        style: form
+      - explode: true
+        in: query
+        name: sort
+        required: false
+        schema:
+          type: string
+        style: form
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UsersResponse'
+          description: List of local users
+        "401":
+          description: No authorization for this endpoint
+      security:
+      - bearerAuth: []
+      tags:
+      - user
+    post:
+      description: |-
+        Create a new user.
+
+        If `uid` is not provided it is automatically filled with the next one available.
+
+        `group` is required if `group_create` is false.
+
+        `password` is required if `password_disabled` is false.
```

Comment:
if you set the `password` you can't really read it back right? I wonder if it makes sense to just do password_disabled by default in terraform, there is no way for terraform to know if password changed 🤷  otherwise you would have to choose to either make password change every-time you do `apply` or ignore it and only set it once.. thats one of the reasons why I think users may not be best resource for terraform... its funny because I ran into similar issue on other terraform provider.. I'm not sure what is the best approach here

Reply:
I guess I'm fine leaving this in the api client, but it may need little more thought on terraform side